File: 백준/Bronze/13458. 시험 감독/시험 감독.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split("\n");

const N = Number(input[0]);
const students = input[1].split(" ").map(Number);
const [B, C] = input[2].split(" ").map(Number);

let answer = 0;

for (const student of students) {
  let remain = student;
  remain -= B;
  answer++;
  if (remain > 0) {
    const t = Math.ceil(remain / C);
    answer += t;
  }
}
console.log(answer);

File: 백준/Bronze/13458. 시험 감독/시험 감독.py
import sys
import math
input = sys.stdin.readline

def solve():
    N = int(input())
    students = list(map(int, input().split()))
    B, C = list(map(int, input().split()))
    
    answer = 0
    for student in students:
        remain = student - B
        answer += 1
        if remain > 0:
            answer += math.ceil(remain / C)
    print(answer)


if __name__ == "__main__":
    solve()